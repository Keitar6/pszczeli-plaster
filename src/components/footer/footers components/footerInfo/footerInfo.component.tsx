import { FC } from "react";

import { TextLink } from "../../../../global.styles";
import {
  InformacjeContainer,
  InformacjeLinks,
  InformacjeTitle
} from "./footerInfo.styles";

export const Info: FC = () => {
  return (
    <InformacjeContainer>
      <InformacjeTitle>Informacje</InformacjeTitle>
      <InformacjeLinks>
        <TextLink>Warunki reklamacji</TextLink>
        <TextLink>Zwroty towaru</TextLink>
        <TextLink>Regulamin</TextLink>
        <TextLink>Polityka cookies</TextLink>
      </InformacjeLinks>
    </InformacjeContainer>
  );
};
