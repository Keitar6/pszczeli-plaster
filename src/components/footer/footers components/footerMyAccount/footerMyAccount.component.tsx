import { FC } from "react";

import { BareLink, TextLink } from "../../../../global.styles";
import {
  MojeKontoContainer,
  MojeKontoLinks,
  MojeKontoTitle
} from "./footerMyAccount.styles";

export const MyAccount: FC = () => {

  return (
    <MojeKontoContainer>
      <MojeKontoTitle>Moje Konto</MojeKontoTitle>
      <MojeKontoLinks>
        <TextLink>
          <BareLink to="/historiaZamowien">Historia zamówień</BareLink>{" "}
        </TextLink>
        <TextLink>
          <BareLink to="/mojeKonto">Moje Konto</BareLink>
        </TextLink>
      </MojeKontoLinks>
    </MojeKontoContainer>
  );
};
