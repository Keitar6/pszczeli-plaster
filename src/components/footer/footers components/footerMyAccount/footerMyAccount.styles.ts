import styled from "styled-components";

export const MojeKontoLinks = styled.div`
  display: flex;
  flex-direction: column;
`;

export const MojeKontoTitle = styled.h4`
  margin: 1rem 0 1rem 0;
`;

export const MojeKontoContainer = styled.div`
  display: flex;
  flex-direction: column;
  align-items: left;
`;
