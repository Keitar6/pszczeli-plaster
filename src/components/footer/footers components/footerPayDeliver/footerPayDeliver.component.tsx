import { FC } from "react";

import { TextLink } from "../../../../global.styles";
import {
  PlatnosciIDostawaContainer,
  PlatnosciIDostawaLinks,
  PlatnosciIDostawaTitle
} from "./footerPayDeliver.styles";

export const PayDeliver: FC = () => {
  return (
    <PlatnosciIDostawaContainer>
      <PlatnosciIDostawaTitle>Płatności i dostawa</PlatnosciIDostawaTitle>
      <PlatnosciIDostawaLinks>
        <TextLink>Czas realizacji zamówień</TextLink>
        <TextLink>Faktury i paragony</TextLink>
        <TextLink>Opcje dostawy</TextLink>
      </PlatnosciIDostawaLinks>
    </PlatnosciIDostawaContainer>
  );
};
