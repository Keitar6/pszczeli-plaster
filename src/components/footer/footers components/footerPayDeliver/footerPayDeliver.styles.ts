import styled from "styled-components";

export const PlatnosciIDostawaLinks = styled.div`
  display: flex;
  flex-direction: column;
`;

export const PlatnosciIDostawaTitle = styled.h4`
  margin: 1rem 0 1rem 0;
`;

export const PlatnosciIDostawaContainer = styled.div`
  display: flex;
  flex-direction: column;
  align-items: left;
`;
