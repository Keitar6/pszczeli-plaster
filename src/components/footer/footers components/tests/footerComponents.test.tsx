import ComponentsRenderer from "react-test-renderer";
import { AboutUs } from "../footerAboutUs/footerAboutUs.component";
import React from "react";
import { PayDeliver } from "../footerPayDeliver/footerPayDeliver.component";
import { Info } from "../footerInfo/footerInfo.component";
import { MyAccount } from "../footerMyAccount/footerMyAccount.component";

describe("Footer", () => {


  test("footerAboutUs", () => {
    const text = ComponentsRenderer.create(<AboutUs></AboutUs>).toJSON();
    expect(text).toMatchSnapshot();
  });
  test("footerInfo", () => {
    const text = ComponentsRenderer.create(<Info></Info>).toJSON();
    expect(text).toMatchSnapshot();
  });
  test("footerMyAccount", () => {
    const text = ComponentsRenderer.create(<MyAccount></MyAccount>).toJSON();
    expect(text).toMatchSnapshot();
  });
  test("footerPayDeliver", () => {
    const text = ComponentsRenderer.create(<PayDeliver></PayDeliver>).toJSON();
    expect(text).toMatchSnapshot();
  });

  //
});
