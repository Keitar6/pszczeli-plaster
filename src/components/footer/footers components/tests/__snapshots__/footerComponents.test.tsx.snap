// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer footerAboutUs 1`] = `
.c3 {
  cursor: pointer;
  position: relative;
}

.c3::before {
  content: "";
  position: absolute;
  width: 100%;
  height: 4px;
  border-radius: 4px;
  background-color: #18272f;
  bottom: 0;
  left: 0;
  -webkit-transform-origin: right;
  -ms-transform-origin: right;
  transform-origin: right;
  -webkit-transform: scaleX(0);
  -ms-transform: scaleX(0);
  transform: scaleX(0);
  -webkit-transition: -webkit-transform 0.3s ease-in-out;
  -webkit-transition: transform 0.3s ease-in-out;
  transition: transform 0.3s ease-in-out;
}

.c3:hover::before {
  -webkit-transform-origin: left;
  -ms-transform-origin: left;
  transform-origin: left;
  -webkit-transform: scaleX(1);
  -ms-transform: scaleX(1);
  transform: scaleX(1);
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c1 {
  margin: 1rem 0 1rem 0;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: left;
  -webkit-box-align: left;
  -ms-flex-align: left;
  align-items: left;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: left;
  -webkit-box-align: left;
  -ms-flex-align: left;
  align-items: left;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  margin: 0.2rem 0 1rem 0;
}

.c5 {
  cursor: pointer;
}

<div
  className="c0"
>
  <h4
    className="c1"
  >
    O Nas
  </h4>
  <div
    className="c2"
  >
    <div
      className="c3"
    >
      O nas
    </div>
    <div
      className="c3"
    >
      Sklepy stacjonarne
    </div>
    <div
      className="c3"
    >
      Współpraca
    </div>
    <div
      className="c3"
    >
      Media społecznościowe
    </div>
    <div
      className="c4"
    >
      <a
        className="c5"
      >
        <span />
      </a>
      <a
        className="c5"
      >
        <span />
      </a>
      <a
        className="c5"
      >
        <span />
      </a>
      <a
        className="c5"
      >
        <span />
      </a>
    </div>
  </div>
</div>
`;

exports[`Footer footerInfo 1`] = `
.c3 {
  cursor: pointer;
  position: relative;
}

.c3::before {
  content: "";
  position: absolute;
  width: 100%;
  height: 4px;
  border-radius: 4px;
  background-color: #18272f;
  bottom: 0;
  left: 0;
  -webkit-transform-origin: right;
  -ms-transform-origin: right;
  transform-origin: right;
  -webkit-transform: scaleX(0);
  -ms-transform: scaleX(0);
  transform: scaleX(0);
  -webkit-transition: -webkit-transform 0.3s ease-in-out;
  -webkit-transition: transform 0.3s ease-in-out;
  transition: transform 0.3s ease-in-out;
}

.c3:hover::before {
  -webkit-transform-origin: left;
  -ms-transform-origin: left;
  transform-origin: left;
  -webkit-transform: scaleX(1);
  -ms-transform: scaleX(1);
  transform: scaleX(1);
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c1 {
  margin: 1rem 0 1rem 0;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: left;
  -webkit-box-align: left;
  -ms-flex-align: left;
  align-items: left;
}

<div
  className="c0"
>
  <h4
    className="c1"
  >
    Informacje
  </h4>
  <div
    className="c2"
  >
    <div
      className="c3"
    >
      Warunki reklamacji
    </div>
    <div
      className="c3"
    >
      Zwroty towaru
    </div>
    <div
      className="c3"
    >
      Regulamin
    </div>
    <div
      className="c3"
    >
      Polityka cookies
    </div>
  </div>
</div>
`;

exports[`Footer footerMyAccount 1`] = `
.c3 {
  cursor: pointer;
  position: relative;
}

.c3::before {
  content: "";
  position: absolute;
  width: 100%;
  height: 4px;
  border-radius: 4px;
  background-color: #18272f;
  bottom: 0;
  left: 0;
  -webkit-transform-origin: right;
  -ms-transform-origin: right;
  transform-origin: right;
  -webkit-transform: scaleX(0);
  -ms-transform: scaleX(0);
  transform: scaleX(0);
  -webkit-transition: -webkit-transform 0.3s ease-in-out;
  -webkit-transition: transform 0.3s ease-in-out;
  transition: transform 0.3s ease-in-out;
}

.c3:hover::before {
  -webkit-transform-origin: left;
  -ms-transform-origin: left;
  transform-origin: left;
  -webkit-transform: scaleX(1);
  -ms-transform: scaleX(1);
  transform: scaleX(1);
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c1 {
  margin: 1rem 0 1rem 0;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: left;
  -webkit-box-align: left;
  -ms-flex-align: left;
  align-items: left;
}

<div
  className="c0"
>
  <h4
    className="c1"
  >
    Moje Konto
  </h4>
  <div
    className="c2"
  >
    <div
      className="c3"
    >
      Historia zamówień
    </div>
    <div
      className="c3"
    >
      Ustawienia Konta
    </div>
  </div>
</div>
`;

exports[`Footer footerPayDeliver 1`] = `
.c3 {
  cursor: pointer;
  position: relative;
}

.c3::before {
  content: "";
  position: absolute;
  width: 100%;
  height: 4px;
  border-radius: 4px;
  background-color: #18272f;
  bottom: 0;
  left: 0;
  -webkit-transform-origin: right;
  -ms-transform-origin: right;
  transform-origin: right;
  -webkit-transform: scaleX(0);
  -ms-transform: scaleX(0);
  transform: scaleX(0);
  -webkit-transition: -webkit-transform 0.3s ease-in-out;
  -webkit-transition: transform 0.3s ease-in-out;
  transition: transform 0.3s ease-in-out;
}

.c3:hover::before {
  -webkit-transform-origin: left;
  -ms-transform-origin: left;
  transform-origin: left;
  -webkit-transform: scaleX(1);
  -ms-transform: scaleX(1);
  transform: scaleX(1);
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c1 {
  margin: 1rem 0 1rem 0;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: left;
  -webkit-box-align: left;
  -ms-flex-align: left;
  align-items: left;
}

<div
  className="c0"
>
  <h4
    className="c1"
  >
    Płatności i dostawa
  </h4>
  <div
    className="c2"
  >
    <div
      className="c3"
    >
      Czas realizacji zamówień
    </div>
    <div
      className="c3"
    >
      Faktury i paragony
    </div>
    <div
      className="c3"
    >
      Opcje dostawy
    </div>
  </div>
</div>
`;
